.github/workflows/e2e_tests.yml

```diff
@@ -138,13 +138,41 @@ jobs:
         run: echo "LTS_CYCLE_FIRST_SDK_REF=release/2.129.0" >> $GITHUB_ENV # this should be updated when a new LTS cycle starts
 
       - name: Get latest release branch
-        shell: bash
+        shell: python
         run: |
-          last_version=$(curl --silent "https://api.github.com/repos/kili-technology/kili-python-sdk/releases/latest" | jq -r .tag_name)
-          echo "Last version: $last_version"
-          IFS=. read -r major minor patch <<< "$last_version"
-          last_minor_version="$major.$minor.0"
-          echo "LASTEST_RELEASE_BRANCH=release/$last_minor_version" >> $GITHUB_ENV
+          import urllib.request
```

Comment:
for long scripts like that, it can be preferrable to keep them in a separate Python file to benefit from the Python dev tooling

Reply:
it's preferable but more prone to bugs

if we put that in a python script, the python script will have to be downloaded, since at this point in the workflow we haven't checkout the repo yet

also, we would have to parse the output of the python script and store it in a bash variable. The parsing of the python script output should be easy, but if we have a bug in the python script, it might not appear in the workflow logs, which makes debugging complicated